recipes/tamper/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "tamper" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/bcgsc/tAMPer/releases/download/v{{ version }}/tAMPer_v{{ version }}.tar.gz
+  sha256: 0c4cfc57a844b89dbc9c020084b5d12b096359a7341350552f5ea707d882486a
+
+build:
+  number: 1
```

I think you need to set the build number to 0.
```suggestion
  number: 0
```